pkg/kinflate/tree/builder_test.go

```diff
@@ -321,8 +329,9 @@ func TestLoadManifestDataFromPath(t *testing.T) {
 	parent1.Packages = []*ManifestData{child1}
 	parent2.Packages = []*ManifestData{child2}
 
+	loader := Loader{FS: fs.MakeRealFS()}
 	expected := grandparent
-	actual, err := loadManifestDataFromPath("testdata/hierarchy")
+	actual, err := loader.loadManifestDataFromPath("testdata/hierarchy")
```

Comment:
Should we test "private" methods?

Reply:
I think so. It doesn't matter if the method is private or not. We need some tests to cover the code.